charts/zenko/values.yaml

```diff
@@ -24,17 +24,56 @@ ingress:
     #   hosts:
     #     - zenko.example.com
 
+global:
+  orbit:
+    enabled: true
+    endpoint: https://api.zenko.io
+    # When 'orbit.enabled' is 'true', these aren't used, please use
+    # https://zenko.io to manage your deployment
+  locationConstraints: {}
+    # awsbackend:
```

Comment:
@ssalaues Do you think we should leave this comments under `locationConstraints`?